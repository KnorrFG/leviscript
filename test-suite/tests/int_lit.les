let n = 20
x{ echo $n }
let n = -20
x{ echo $n }
let n = 20_000
x{ echo $n }
let n = 0xAB
x{ echo $n }
let n = -0xAB
x{ echo $n }
