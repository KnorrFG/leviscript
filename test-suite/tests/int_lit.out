20
-20
20000
171
-171
